app/views/wizard/steps/duty/calculations/_option_row.html.erb

```diff
@@ -1,5 +1,5 @@
 <tr class="govuk-table__row">
-  <% row.each do |value| %>
-    <td class="govuk-table__cell govuk-body-s"><%= value %></td>
-  <% end %>
+    <td class="govuk-table__cell govuk-body-s"><%= row[0] %></td>
+    <td class="govuk-table__cell govuk-body-s"><%= row[1] %></td>
+    <td class="govuk-table__cell govuk-body-s govuk-table__cell--numeric"><%= row[2] %></td>
```

Comment:
This right-aligns govuk table column:

![Screenshot from 2021-04-13 09-27-08](https://user-images.githubusercontent.com/8156884/114524924-8dbf6180-9c3d-11eb-9a63-5b8fb55fbefa.png)